### DISCUSSION TITLE
リポジトリエラー確認したい！

### DISCUSSION THREAD

**Discussion by urassh** [ProgateHackathon/Gitagram#106] [Q&A] [answered]
@am2525nyan 
このエラー確かに、確認できたのだけれど、リロードしたら、新しく追加したリポジトリ追加されてたんよね。
どういう時に起きるか確認しておきたい。

https://github.com/ProgateHackathon/Gitagram/pull/101#pullrequestreview-2150230439

**Comment by am2525nyan** [comment]
@urassh 
追加したリポジトリ表示されました！ワンチャン私が気付いてなかった説あります。。。。

**Reply by urassh** [reply]
@am2525nyan 
エラーの内容的に、Firebaseから届いたデータをResponseに変換するのがうまくいってないっって言われてる感じがする。
どのタイミングで起きているか判明 or 原因までわかったら、ここで議論していこー！

**Reply by am2525nyan** [reply]
OKですー！！！！

**Comment by am2525nyan** [comment, marked as answer]
初ディスカッションデビュー

**Reply by urassh** [reply]
いぇー！🤟

**Comment by am2525nyan** [comment]
` return HashTag(id: HashTag.HashTagID(id: id), name: name, color: color)`
 ここで起きてそう感あります

**Comment by am2525nyan** [comment]
`Error decoding city: valueNotFound(Swift.KeyedDecodingContainer<Gitagram.HashTagResponse.(unknown context at $1051082bc).CodingKeys>, Swift.DecodingError.Context(codingPath: [], debugDescription: "Cannot get keyed decoding container -- found null value instead.", underlyingError: nil))`

これが6回起きてる認識であってます？
